src/library/scala/sys/BooleanProp.scala

```diff
@@ -63,12 +63,13 @@ object BooleanProp {
   def valueIsTrue[T](key: String): BooleanProp = new BooleanPropImpl(key, _.toLowerCase == "true")
 
   /** As an alternative, this method creates a BooleanProp which is true
-   *  if the key exists in the map.  This way -Dfoo.bar is enough to be
-   *  considered true.
+   *  if the key exists in the map and is not assigned a value other than "true",
```

Comment:
The double negation could do with some De Morgan. `!(assigned !"true")` --> `"The key exists in the map and is either the empty string or `"true"`"`.